netbox/extras/forms/reports.py

```diff
@@ -15,3 +15,8 @@ class ReportForm(BootstrapMixin, forms.Form):
         label=_("Schedule at"),
         help_text=_("Schedule execution of report to a set time"),
     )
+    interval = forms.IntegerField(
```

Comment:
Setting the interval in minutes is maybe not the most userfriendly way. Supporting something like m/h/d suffixes might be an option, shouldn't be too involved.